src/audio/aria/aria_hifi5.c

```diff
@@ -57,17 +57,19 @@ inline void aria_algo_calc_gain(struct aria_data *cd, size_t gain_idx,
 		inu = AE_LA128_PP(in);
 		for (i = 0; i < n; i += 4) {
 			AE_LA32X2X2_IP(in_sample, in_sample1, inu, in);
-			in_sample = AE_MAXABS32S(in_sample, in_sample1);
-			max_data = AE_MAXABS32S(max_data, in_sample);
+			max_data = AE_MAXABS32S(max_data, AE_SLAI32(in_sample1, 8));
```

Comment:
is this intentional? So `in_sample` below is used as set in line 59 above?

Reply:
Yes, it's intentional. I think this is more clear, and no need to have left shift for both first AE_MAXABS32S input arguments.

Reply:
Also, now all (generic, hifi3/4, hifi5) versions of Aria are bit exact with the input that I use.